packages/terra-menu/src/MenuItem.jsx

```diff
@@ -261,7 +261,7 @@ class MenuItem extends React.Component {
     } = this.context;
     const attributes = { ...customProps };
 
-    attributes.tabIndex = isDisabled ? '-1' : '0';
+    attributes.tabIndex = '0';
```

Comment:
since there is no condition expression involved we can move this to DOM element

Reply:
moved to DOM in https://github.com/cerner/terra-framework/pull/1655/commits/5f43a1703779ea5657298ce87536f0a641f0a513